gamearea.py

```diff
@@ -441,8 +442,21 @@ def __draw_end_message(self, context):
             else:
                 message = _("You should have selected an option")
                 self.win = False
-            if self.puzzle_count < self.max_puzzle_count:
-                y = self.show_message(context, message, 64)
+
+        elif self.level_data["type"] == GameType.CHOOSE:
+            if self.selected_option is not None:
+                if self.selected_option.cat_id == self.odd_cat_id:
+                    message = _("You Choose correctly!")
+                    self.win = True
+                else:
+                    message = _("You Choose wrong")
+                    self.win = False
+            else:
+                message = _("You should have chosen an option")
```

Comment:
choose should be small letters and it should be in past tense as the time this message shows, you're expected to have made your choices.

Reply:
This still uses "Choose" and not "chose".

Reply:
Seen your change.